packages/config/config/devices/0x019b/zm_single_relay_16.json

```
@@ -0,0 +1,228 @@
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Load Limit",
+			"description": "1-16 A",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 16,
+			"defaultValue": 16,
+			"unsigned": true
+		},
+		"2": {
+			"label": "Power Shutdown Actions",
+			"description": "0: Disabled, 1-32767: minute(s) delay before attempting to turn back on",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 32767,
+			"defaultValue": 20,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Disabled and will not retry. User needs to manually turn on afterwards. If temperature overload is on, device will not turn on until device has cooled down",
+					"value": 0
+				}
+			]
+		},
+		"3": {
+			"label": "Switch Type",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Momentary switch",
+					"value": 0
+				},
+				{
+					"label": "Toggle switch",
+					"value": 1
+				}
+			]
+		},
+		"4": {
+			"label": "S1/Button Operation",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Button turns load on/off and sends meter report + relay status",
+					"value": 0
+				},
+				{
+					"label": "Button sends meter report + relay status, load can be only controlled wirelessly",
+					"value": 1
+				}
+			]
+		},
+		"5": {
+			"label": "Scene Notifications",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Sends scene controller for s2. s1 disabled",
+					"value": 0
+				},
+				{
+					"label": "Sends scene controller for s1. s2 disabled",
+					"value": 1
+				},
+				{
+					"label": "Sends scene controller for s1 and s2",
+					"value": 2
+				},
+				{
+					"label": "Scene controller deactivated",
+					"value": 3
+				}
+			]
+		},
+		"6": {
+			"label": "Restore Power Level",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 2,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Always off on restored power",
+					"value": 0
+				},
+				{
+					"label": "Always on on restored power",
+					"value": 1
+				},
+				{
+					"label": "Restore last state on restored power (default)",
+					"value": 2
+				}
+			]
+		},
+		"7": {
+			"label": "Automatic Turn Off",
+			"description": "Time to turn relay off after having been turned on (1-86400 seconds)",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 86400,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Auto off disabled",
+					"value": 0
+				}
+			]
+		},
+		"8": {
+			"label": "Automatic Turn On",
+			"description": "Time to turn relay on after having been turned off (1-86400 seconds)",
```

```suggestion
			"label": "Auto-On Delay",
			"description": "Time to turn relay on after having been turned off",
			"unit": "seconds",
```
IMO here the description makes sense because it is less common to have this functionality